csharp/src/Ice/OutgoingResponseFrame.cs

```diff
@@ -174,6 +182,11 @@ public static OutgoingResponseFrame WithVoidReturnValue(Current current)
                         var replyStatus = (ReplyStatus)response.Payload[0];
                         if (replyStatus == ReplyStatus.UserException)
                         {
+                            // We are forwarding an ice1 user exception using ice2. We create an ice2 encapsulation
```

Comment:
For consistency with the previous comments, I would use:
We are forwarding a user exception encoded using 1.1 and received over ice1 to ice2.